Source/FieldSolver/ImplicitSolvers/ImplicitSolver.cpp

```diff
@@ -726,7 +728,7 @@ void ImplicitSolver::PreRHSOp ( const amrex::Real  a_cur_time,
 
     if (m_use_mass_matrices && !a_from_jacobian) { // Called from non-linear stage of JFNK and using mass matrices
         options.deposit_mass_matrices = true;
-        m_WarpX->PushParticlesandDeposit(a_cur_time, skip_current, &options);
+        m_WarpX->PushParticlesandDeposit(a_cur_time, skip_current, position_push_type, momentum_push_type, &options);
```

Comment:
I would have a preference for making this explicit:
```suggestion
        m_WarpX->PushParticlesandDeposit(a_cur_time, skip_current, PositionPushType::Full, MomentumPushType::Full, &options);
```

Reply:
Done in [b729f74](https://github.com/BLAST-WarpX/warpx/pull/5955/commits/b729f74f848bb42f6b1a1659487448db823032dd).